tutorials/hyperformula-integration.html

```diff
@@ -0,0 +1,257 @@
+          ["184", "630", "0.19", "6", "=A82*(B82*C82)+D82"], ["27", "165", "0.51", "13", "=A83*(B83*C83)+D83"], ["186", "417", "0.85", "15", "=A84*(B84*C84)+D84"], ["20", "741", "0.76", "11", "=A85*(B85*C85)+D85"], ["153", "640", "0.28", "6", "=A86*(B86*C86)+D86"], ["161", "542", "0.43", "14", "=A87*(B87*C87)+D87"], ["98", "344", "0.77", "6", "=A88*(B88*C88)+D88"], ["30", "400", "0.71", "13", "=A89*(B89*C89)+D89"], ["73", "91", "0.78", "8", "=A90*(B90*C90)+D90"], ["158", "72", "0.66", "12", "=A91*(B91*C91)+D91"],
+          ["122", "35", "0.32", "15", "=A92*(B92*C92)+D92"], ["33", "99", "0.92", "9", "=A93*(B93*C93)+D93"], ["107", "538", "0.75", "10", "=A94*(B94*C94)+D94"], ["15", "563", "0.32", "6", "=A95*(B95*C95)+D95"], ["168", "572", "0.71", "6", "=A96*(B96*C96)+D96"], ["135", "217", "0.49", "12", "=A97*(B97*C97)+D97"], ["11", "595", "0.03", "13", "=A98*(B98*C98)+D98"], ["41", "739", "0.88", "11", "=A99*(B99*C99)+D99"], ["144", "289", "0.87", "13", "=A100*(B100*C100)+D100"],
+          ["Sum", "Average", "Average", "Sum", "Sum"],
+          ["=SUM(A2:A100)", "=AVERAGE(B2:B100)", "=AVERAGE(C2:C100)", "=SUM(D2:D100)", "=SUM(E2:E100)"]
+        ];
+
+        var container = document.getElementById('example-data-grid');
+        new Handsontable(container, {
+          data: data,
+          formulas: {
+            engine: HyperFormula,
+          },
+          colHeaders: ["Qty", "Unit price", "Discount", "Freight", "Total due (fx)"],
+          fixedRowsBottom: 2,
+          height: 500
+        });</script>
+    </div>
+
+    <h3 id="initialization-methods">Initialization methods</h3>
+    <p>There're multiple ways of initializing the plugin, you can select the most convenient one depending on your use case.</p>
+
+    <p>In all cases it is required to either pass in the <code>HyperFormula</code> object or a HyperFormula instance:</p>
+    <pre><code class="hljs lang-javascript">import { HyperFormula } from 'hyperformula';</code></pre>
+    <p>There are also other installation methods available, check out <a href="https://handsontable.github.io/hyperformula/guide/client-side-installation.html">HyperFormula's installation documentation</a>.</p>
+
+
+    <!-- todo should we write demos for these? -->
+    <h4 id="passing-hf-to-hot">Passing the HyperFormula class/instance to Handsontable</h4>
+    <pre><code class="hljs lang-javascript">{
+  formulas: {
+    engine: HyperFormula,
+    // [plugin configuration]
+  }
+}</code></pre>
+    <p>or</p>
+    <pre><code class="hljs lang-javascript">{
+  formulas: {
+    engine: {
+      hyperformula: HyperFormula, // or `engine: hyperformulaInstance`
+    },
+    // [plugin configuration]
+  }
```

Comment:
Here I'd add some config examples from HyperFormula. It's easier to learn from examples. So something like:
```suggestion
  formulas: {
    engine: {
      hyperformula: HyperFormula, // or `engine: hyperformulaInstance`
      leapYear1900: false,
      binarySearchThreshold: 15,
      // ... and more configuration options. See https://handsontable.github.io/hyperformula/api/interfaces/configparams.html#number 
    },
    // [plugin configuration]
  }
```